app/app.py

```diff
@@ -40,11 +42,14 @@ def linchpin_list_workspace():
                 workspace_array.append(workspace_dict)
         return Response(json.dumps(workspace_array), status=200, mimetype='application/json')
     except Exception as e:
-        print(e)
+        app.logger.error(e)
         return jsonify(status=409, message=str(e))
 
 
 if __name__ == "__main__":
+    handler = RotatingFileHandler('restylinchpin.log', maxBytes=10000, backupCount=1)
```

Comment:
@mansikulkarni96 
We should not hardcode log file and it should always be configurable. 
Please assign restylinchpin.log to variable and load it from the config.yml file which you have created

Reply:
ok will do that thanks!